tests/jballerina-unit-test/src/test/java/org/ballerinalang/nativeimpl/jvm/servicetests/ServiceValue.java

```diff
@@ -165,4 +176,15 @@ public static BArray getServicePath() {
     public static BMap getAnnotationsAtServiceAttach() {
         return ServiceValue.annotationMap;
     }
+
+    public static BValue getAnnotMap(BObject service, BString method, ArrayValue path, BString annotName) {
```

Comment:
Is the name correct here? The annotation may not be a map, right?

Reply:
This method seems to be used only in tests? I'm not sure if we should be introducing a method just for tests. Can't we extract this information using `getResourceAnnotation`?

Reply:
Missed that this was a test class, my bad. Shall we rename the method though?